db/mongoose.js

```diff
@@ -146,6 +166,27 @@ async function runQuery() {
   return [formattedCurrentTime, priceResponse];
 }
 
+async function fetchIndex() {
+  const formattedCurrentTime = await formatCurrentTime();
+
+  let priceResponse;
+
+  try {
+    const doc = new GoogleSpreadsheet("1ghUzXmKIcxJgfLIymZZ373iUPpVAfWfqZcdv4hCxhKk");
+    await doc.useApiKey("AIzaSyA67a7rX10nA2qAoW5bCvOYXqFkszMbEU0");
```

Comment:
this should go in .env
```suggestion
    await doc.useApiKey(process.env.GAPI_KEY);
```